tools/cpp/windows_cc_toolchain_config.bzl

```diff
@@ -377,6 +377,7 @@ def _impl(ctx):
 
         compiler_param_file_feature = feature(
             name = "compiler_param_file",
```

Comment:
is `linker_param_file` enabled as well?

Reply:
Having a quick look it seems nothing has changed with respect to the feature linker_param_file in a long time. You can try passing --features=linker_param_file, if that works great. It still wouldn't explain why you see this difference with respect to 5.4.

Let's just focus on compiler_param_file feature here and let's continue on https://github.com/bazelbuild/bazel/issues/17068.